src/pages/auth/login.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+
+const login = () => {
+  return (
+    <div>
+      <main className="auth-container">
+        <form className="card-vertical signup-form">
+          <h3>Login</h3>
+
+          <div className="input-group-parent">
+            <div className="input-group">
+              <label>Email address</label>
+              <input
+                className="input-box"
+                type="email"
+                placeholder="vishal@mail.com"
+              />
+            </div>
+
+            <div className="input-group">
+              <label> Password </label>
+              <input
+                className="input-box"
+                type="password"
+                placeholder="******"
+              />
+            </div>
+
+            <div className="select-box">
+              <input type="checkbox" name="accept" />
```

Comment:
Put input inside label so that it will work on clicking on the text as well

Reply:
Thanks for picking that out.